apps/server/src/modules/learnroom/service/common-cartridge-export.service.spec.ts

```diff
@@ -133,6 +133,14 @@ describe('CommonCartridgeExportService', () => {
 				});
 			});
 
+			it("shouldn't add lessons if topics array is empty", async () => {
+				const { archive, lessons } = await setup(false);
+
+				lessons.forEach((lesson) => {
+					expect(getFileContent(archive, 'imsmanifest.xml')).not.toContain(createXmlString('title', lesson.name));
+				});
+			});
```

Comment:
This must be placed in a new scenario and the setup function does not take any parameters.